static/app/components/replays/videoReplayerWithInteractions.tsx

```diff
@@ -121,6 +124,22 @@ export class VideoReplayerWithInteractions {
       }
     });
 
+    // log instances where we have a pointer touchStart without a touchEnd
+    const touchEvents = eventsWithSnapshots.filter(
+      (e: eventWithTime) => isTouchEnd(e) || isTouchStart(e)
+    );
+    const grouped = Object.groupBy(touchEvents, (t: any) => t.data.pointerId);
+    Object.values(grouped).forEach(t => {
+      if (t?.length !== 2) {
```

Comment:
Prob good to log the value of `t` too. And if a number, the value. 
JS rusty but it could be `null` `undefined`, `1`, `3` or more right?

Reply:
so `t` is something like this:

<img width="308" alt="SCR-20240815-mkvy" src="https://github.com/user-attachments/assets/27f242ac-609d-48be-a051-d5690bd1a6f7">

but i'm not sure any of this information is useful from a debugging perspective. we can also only log primitives as tags, not objects :(

the value of `grouped` is something like this: an object of arrays, the arrays contain items similar to `t`

<img width="227" alt="SCR-20240815-mksp" src="https://github.com/user-attachments/assets/d2faf34f-fe6f-492d-b58e-5138c96ab7ec">

the arrays will always be at least length 1 because that's what we filter for up above!

Reply:
You could dump t in extra. `Sentry.withScope(s => { 
s.setExtra(t);
Sentry.captureMessage(..);
});

That's 'cheap' since it's not indexed/searchable.
But if you're confident the whole thing isn't helpful, you might want to just log the `type` since it's useful to know if we're getting `null` at least, no? That's what I read when I see the `?` in there.

Reply:
ahh i see. added a tag for the type of `t`